projects/storefrontstyles/scss/components/layout/_storefront.scss

```diff
@@ -1,10 +1,19 @@
 @import './progress-bar';
 
 %cx-storefront {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
```

Comment:
we use evergreen browsers, so no need to have additional 100% for min-height

Reply:
old habit ;)